package.json

```diff
@@ -22,7 +22,7 @@
     "bfx-api-node-core": "^1.5.5",
     "bfx-api-node-plugin-wd": "^1.0.4",
     "bfx-api-node-rest": "^4.3.1",
-    "bfx-hf-algo": "git+https://github.com/bitfinexcom/bfx-hf-algo.git#v4.7.3",
+    "bfx-hf-algo": "git+https://github.com/avsek477/bfx-hf-algo.git#add-algo-id-in-ao-events",
```

Comment:
needs update when this [PR](https://github.com/bitfinexcom/bfx-hf-algo/pull/193) is merged

Reply:
```suggestion
    "bfx-hf-algo": "git+https://github.com/bitfinexcom/bfx-hf-algo.git#v4.8.0",
```